bip-0039/german.txt

```diff
@@ -0,0 +1,2048 @@
+ABDRUCK
+ABEND
+ABFALL
+ABFLUG
+ABFOLGE
+ABGAS
+ABGLEICH
+ABHANG
+ABITUR
+ABKOMMEN
+ABMARSCH
+ABONNENT
+ABREISE
+ABSCHIED
+ABSEITS
+ABSICHT
+ABSPANN
+ABSTRICH
+ABTEIL
+ABWASCH
+ABZWEIG
+ACHSEL
+ACHTUNG
+ACKERBAU
+ADAPTER
+ADDITION
+ADJEKTIV
+ADJUDANT
```

Comment:
Spelled w/ `t` instead of `d`.

```suggestion
ADJUTANT
```
I would have written it the same way, wasn't sure though, googled it and found it's spelled differently.
- https://www.google.com/search?q=adjutant
- https://de.wikipedia.org/wiki/Adjutant

Reply:
Will be accepted of course. Unfortunately it mainly shows that the spell checker used didn't work properly - I now quickly tested some other spell checkers and interestingly they come to different conclusions for certain words. Will evaluate later which spell checker works best for our requirements.

Reply:
Went through the rest of the list word by word @SebastianFloKa, didn't find anything else, thinking this one was an outlier.

Reply:
@thomasklemm thanks a lot for your effort, really appreciated. 
I used a different spellchecker now and one more typo is "Avocado" which will be removed.
We can put all of your highlighted words on a "preliminary replacement list". Means I'll not respond to each comment.  Below will then be posted a list of optional words in order to decide which ones should be replaced or not.

Reply:
Word replaced